install.sh

```diff
@@ -195,7 +195,11 @@ fi
 
 download_libs
 # mind the gap (trailing space)
-sha256sum $cachedir/orctlibs.zip | cut -f1 -d\  > $libVFile
+if [[ `uname` == "Darwin" ]]; then
```

Comment:
[$(...) is preferred over backticks](http://mywiki.wooledge.org/BashFAQ/082), and would it maybe be better to look for the actual tool instead?

Here is an example on how to do this the correct way: http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script

We could implement it as such:
```bash
function checksum () {
  if hash sha256sum 2>/dev/null; then
    sha256sum "$1" | cut -f1 -d\ 
  else
    shasum -a 256 "$1" | cut -f1 -d\ 
  fi
}

checksum "$cachedir/orctlibs.zip" > $libVFile
```